flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/CreditBasedClientHandler.java

```diff
@@ -37,18 +43,29 @@
 
 import java.io.IOException;
 import java.net.SocketAddress;
+import java.util.ArrayDeque;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicReference;
 
+/**
+ * Channel handler to read {@link BufferResponse} and {@link ErrorResponse} messages from the
+ * producer, to write and flush {@link AddCredit} message for the producer.
+ */
 class CreditBasedClientHandler extends ChannelInboundHandlerAdapter {
 
 	private static final Logger LOG = LoggerFactory.getLogger(CreditBasedClientHandler.class);
 
+	/** Channels, which already requested partitions from the producers. */
```

Comment:
was this addressed?

Reply:
In this PR I add the new `inputChannelsWithCredit` queue and comment it. In order to contrast 
with previous `inputChannels`, I also add comment for it.  As you said, it can also be done in last commit.

Reply:
I also put this comment and the above comment in last commit and rebase the code today.